hack/choco/tools/chocolateyinstall.ps1

```diff
@@ -93,9 +99,13 @@ function Install-TanzuEnvironment {
 
     # The & allows execution of a binary stored in a variable.
     Write-Host "  - Initializing Tanzu configuration" -ForegroundColor Cyan
-    & $tanzuExe init | Out-Null
-    & $tanzuExe plugin repo add --name tce --gcp-bucket-name tce-tanzu-cli-plugins --gcp-root-path artifacts
-    & $tanzuExe plugin repo add --name core-admin --gcp-bucket-name tce-tanzu-cli-framework-admin --gcp-root-path artifacts-admin
```

Comment:
We don't wanna add the TCE and core-admin plugin repositories after `tanzu init`?